doc/user/content/get-started/_index.md

```diff
@@ -11,14 +11,12 @@ menu:
     weight: 5
 ---
 
-Materialize is the Operational Data Warehouse that delivers the speed of
+Materialize is the Cloud Operational Data Store that delivers the speed of
 streaming with the ease of a data warehouse. With Materialize, organizations can
-operate on real-time data just by using SQL.
+use SQL to transform, deliver, and act on fast-changing data.
 
-If you need to speed up queries that run frequently, or trigger actions as
-soon as events happen, Materialize is a good fit. Rather than recalculate
-results from scratch, or serve stale cached results, Materialize continually
-ingests data and keeps results up-to-date as new data arrives.
+To keeps results up-to-date as new data arrives, Materialize incrementally updates results as it ingests data rather than recalculating results
+from scratch.
```

Comment:
```suggestion
To keeps results up-to-date as new data arrives, Materialize incrementally
updates results as it ingests data rather than recalculating results from
scratch.
```